pkg/util/common.go

```diff
@@ -200,6 +200,7 @@ func CreateHTTPTransport() *http.Transport {
 
 	transport := &http.Transport{
 		TLSClientConfig: tlsConfig,
+		IdleConnTimeout: 90 * time.Second,
```

Comment:
what about `MaxIdleConns`, `TLSHandshakeTimeout`, `ExpectContinueTimeout` and `DialContext`. Do we want to have the same value their also ?

Reply:
We set `http.Client.Timeout` which is a timeout for the whole request everywhere we use this transport. That said I don't see anything preventing us from copying [Golang's default transport](https://github.com/golang/go/blob/go1.10.2/src/net/http/transport.go#L40-L51) completely.

Reply:
It probably makes sense to copy those as well, as long as they don't make the `forwarder_timeout` setting on the client meaningless